Gcpe.Hub.API/Controllers/ActivitiesController.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoMapper;
+using Gcpe.Hub.API.ViewModels;
+using Gcpe.Hub.Data.Entity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+namespace Gcpe.Hub.API.Controllers
+{
+    // TODO: Re-enable this ==> [Authorize]
+    [Route("api/[Controller]")]
+    [ApiController]
+    [Produces("application/json")]
+    public class ActivitiesController : ControllerBase
+    {
+        private readonly HubDbContext dbContext;
+        private readonly ILogger<ActivitiesController> logger;
+        private readonly IMapper mapper;
+
+        public ActivitiesController(HubDbContext dbContext,
+            ILogger<ActivitiesController> logger,
+            IMapper mapper)
+        {
+            this.dbContext = dbContext;
```

Comment:
I'm not 100% sure, but I think the convention is to use an underscore for private fields, so you can avoid having to prefix them with "this" in C#. It's just less mental overhead, but more of a nitpick.

Reply:
We mainly use the Microsoft convention:
https://stackoverflow.com/questions/743808/c-sharp-coding-standards-for-private-member-variables